### ISSUE TITLE
Add support the MAP file output by the Z80ASM from Z88DK

### ISSUE THREAD

**Issue by MBilderbeek** [openMSX/openMSX#1747]
As suggested on Discord: please add support for this symbol format.

> The MAP file output by the Z80ASM that comes with the Z88DK is in this format, and I'm sure many people use the Z88DK, so it would be great if the openMSX-debugger side could support this format. thank you.

```
CHGMOD                          = $005F ; const, local, , sample025, , sample025.asm:2
SETGRP                          = $007E ; const, local, , sample025, , sample025.asm:3
ERAFNK                          = $00CC ; const, local, , sample025, , sample025.asm:4
LINWID                          = $F3AF ; const, local, , sample025, , sample025.asm:7
RG0SAV                          = $F3DF ; const, local, , sample025, , sample025.asm:8
FORCLR                          = $F3E9 ; const, local, , sample025, , sample025.asm:9
```

**Comment by sailorman-msx** [comment]
I have previously modified the openMSX-debugger source to be compatible with Z80ASM on my own.
The source code modified at that time is attached.
[openMSXdebugger_patch_for_z80asm.zip](https://github.com/user-attachments/files/16971229/openMSXdebugger_patch_for_z80asm.zip)

**Comment by sailorman-msx** [comment]
Symbol map file sample is here.
regards.
[map_file_sample_z80asm_format.zip](https://github.com/user-attachments/files/16971244/map_file_sample_z80asm_format.zip)

**Comment by m9710797** [comment]
I've implemented it in revision beaf290e27fb8c3fc5710102e1d7d6a1aea7407f.

Next time don't keep such changes for yourself, then we could have immediately ported the format to the new debugger ;-)
